examples/src/main/java/tachyon/examples/BasicRawTableOperations.java

```diff
@@ -87,17 +88,14 @@ private boolean read(TachyonFS tachyonClient) throws IOException {
     for (int column = 0; column < COLS; column ++) {
       RawColumn rawColumn = rawTable.getRawColumn(column);
       TachyonFile tFile = rawColumn.getPartition(0);
-
-      TachyonByteBuffer buf = tFile.readByteBuffer(0);
-      if (buf == null) {
-        tFile.recache();
-        buf = tFile.readByteBuffer(0);
-      }
-      buf.mData.order(ByteOrder.nativeOrder());
+      InStream is = tFile.getInStream(ReadType.CACHE);
+      ByteBuffer buf = ByteBuffer.allocate((int) tFile.getBlockSizeByte());
+      is.read(buf.array());
+      buf.order(ByteOrder.nativeOrder());
```

Comment:
Just tried to keep consistent with original structure. If there was `nativeOrder` in original code, but missing in diff, I will restore them. Let me double check.